flocker/volume/schema/endpoints.yml

```diff
@@ -0,0 +1,11 @@
+$schema: http://json-schema.org/draft-04/schema#
+id: http://api.clusterhq.com/v1/endpoints.json
+definitions:
+  versions:
```

Comment:
This isn't really tested, i.e. we should have tests validating the schema does what we think it does. We had some in HybridCluster you can use as basis.